### PR TITLE
Modify cpu affinity related VM configs

### PR DESCRIPTION
**Pull request by hky1999**

### CODE REVIEW

**Comment by hky1999** [comment]
maybe arm_vcpu and riscv_vcpu should be updated first.

**Comment by aarkegz** [comment]
> maybe arm_vcpu and riscv_vcpu should be updated first.

Yes, update them first

File: src/config.rs [hunk 1]
```diff
@@ -148,23 +148,30 @@ impl AxVMConfig {
...
     pub size: usize,
+    /// The mappings flags of the memory region, refers to `MappingFlags` provided by `axaddrspace`.
     pub flags: usize,
 }
 
-/// The configuration of axvm crate. It's not used yet, may be used in the future.
+/// The configuration structure for the guest VM serialized from a toml file provided by user,
+/// and then converted to `AxVMConfig` for the VM creation.
 #[derive(Debug, Default, Clone, serde::Serialize, serde::Deserialize)]
 pub struct AxVMCrateConfig {
```
Comment (aarkegz) [inline src/config.rs:176]: Maybe we can merge this struct into `AxVMConfig`?

File: src/vm.rs [hunk 2]
```diff
@@ -231,7 +244,7 @@ impl<H: AxVMHal> AxVM<H> {
                     addr,
                     width,
                     reg,
-                    reg_width,
+                    reg_width: _,
```
Comment (aarkegz) [inline src/vm.rs:247]: `..`.

**Review by aarkegz — APPROVED** [review]

**Pull request merged**